libpod/container_internal.go

```diff
@@ -1022,7 +1023,7 @@ func (c *Container) restartWithTimeout(ctx context.Context, timeout uint) (err e
 			return err
 		}
 	}
-
+	defer c.newContainerEvent(events.Restart)
```

Comment:
Do we want a dedicated restart event? I'm fine having restart generate separate Stop and Start events

Reply:
I agree

Reply:
docker has restart, seems valid... only we consider restart to be stop/start a user wouldnt

Reply:
Hm. Can we keep restart and have it also log the start/stop that happen?